src/backend/web/routes/posts.js

```diff
@@ -112,4 +112,8 @@ posts.get('/:id', async (req, res) => {
   }
 });
 
+posts.get('/allPosts', (req, res) => {
+  res.send(getPostsCount());
```

Comment:
I think [this line](https://github.com/Seneca-CDOT/telescope/blob/baf12536796cda86f630844fe2e652e3b4f97e1b/src/backend/web/routes/posts.js#L58) will save you some work

Reply:
Thanks that saved me a lot